conan/cli/formatters/audit/vulnerabilities.py

```diff
@@ -12,6 +13,11 @@
 }
 
 
+def _raise_if_severity_exceeded(result: tuple[dict, bool]):
```

Comment:
I think it would be better to pass the response dict here directly, all formatters use it